php_phongo.c

```diff
@@ -2228,60 +2259,60 @@ static mongoc_client_t *php_phongo_make_mongo_client(const mongoc_uri_t *uri TSR
 	bson_version = "bundled";
 #endif
 
-	MONGOC_DEBUG("Creating Manager, phongo-%s[%s] - mongoc-%s(%s), libbson-%s(%s), php-%s",
+	MONGOC_DEBUG(
+		"Creating Manager, phongo-%s[%s] - mongoc-%s(%s), libbson-%s(%s), php-%s",
 		PHP_MONGODB_VERSION,
 		PHP_MONGODB_STABILITY,
 		MONGOC_VERSION_S,
 		mongoc_version,
 		BSON_VERSION_S,
 		bson_version,
-		PHP_VERSION
-	);
+		PHP_VERSION);
 
 	return mongoc_client_new_from_uri(uri);
 } /* }}} */
 
-static void php_phongo_persist_client(const char *hash, size_t hash_len, mongoc_client_t *client TSRMLS_DC)
+static void php_phongo_persist_client(const char* hash, size_t hash_len, mongoc_client_t* client TSRMLS_DC)
 {
-	php_phongo_pclient_t *pclient = (php_phongo_pclient_t *) pecalloc(1, sizeof(php_phongo_pclient_t), 1);
+	php_phongo_pclient_t* pclient = (php_phongo_pclient_t*)pecalloc(1, sizeof(php_phongo_pclient_t), 1);
 
-	pclient->pid = (int) getpid();
+	pclient->pid    = (int)getpid();
```

Comment:
Do you see a benefit in leaving whitespace after a cast? I think it does help the function or identifier stand out more.